preview_generator/preview/builder/document_generic.py

```diff
@@ -46,8 +46,8 @@ def build_jpeg_preview(
         preview_name: str,
         cache_path: str,
         page_id: int,
-        extension: str = ".jpg",
-        size: utils.ImgDims = None,
```

Comment:
same, as other, why remove default parameters heres ?

Reply:
same, as other, see previous comment.

Reply:
We could first merge a PR that make them really optional, so in this one we keep them optional?